app/data/locales.json

```diff
@@ -82,5 +82,12 @@
     "value": "sv",
     "key": "i18n.lang.sv",
     "level": 1
-  }
+  },
+  {
+    "label": "Русский",
+    "name": "Russian",
+    "value": "ru",
+    "key": "i18n.lang.ru",
+    "level": 3
```

Comment:
I think the first step is to review the translations and get this merged. Then, we can think about making it a level 3 language. Can you change this to level 2 for now?